woocommerce/woocommerce
# [Settings] Import Site Editor components from node_modules

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Related to https://github.com/woocommerce/woocommerce/issues/52520

This pull request gathers WP Core's Site Editor components directly from the NPM module. This is a similar tactic used by Customize Your Store.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Turn on the feature flag `settings`, Tools > WCA Test Helper > Features.
2. Visit the new Settings, WooCommerce > Settings. You should a message that Gutenberg must be present.
8. Since we are unlocking the same packages from WP used in the Customizer, we need to make sure the Customizer still functions, WooCommerce > Home > Task List > Customize your store
9. Install/Activate Gutenberg, see the new Settings screen load with a Sidebar and placeholder content
10. Check that Customizer is still functioning



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
